build_all_images.sh

```diff
@@ -67,7 +67,12 @@ function build_bundle_image {
 
 	if [ -z "${bundle_url}" ]
 	then
-		bundle_url="releases-cdn.liferay.com/dxp/${version}/"$(curl -fsSL "https://releases-cdn.liferay.com/dxp/${version}/.lfrrelease-tomcat-bundle")
+		if [[ "${version}" == *q1* ]] && [[ "$(echo "${version}" | cut -d '.' -f 1)" -ge 2025 ]]
+		then
+			bundle_url="releases-cdn.liferay.com/dxp/${version}-lts/"$(curl -fsSL "https://releases-cdn.liferay.com/dxp/${version}-lts/.lfrrelease-tomcat-bundle")
+		else
+			bundle_url="releases-cdn.liferay.com/dxp/${version}/"$(curl -fsSL "https://releases-cdn.liferay.com/dxp/${version}/.lfrrelease-tomcat-bundle")
+		fi
```

Comment:
@davisaints, can you please revert this commit and also all commits from https://liferay.atlassian.net/browse/LPD-48381 and try to build the Docker image adding to the `bundles.yml` only `2025.q1.0-lts` instead of `2025.q1.0`?

Last week, I was talking to Nikki about the https://liferay.atlassian.net/browse/LPD-49368 and I believe that, for now, this new suggestion above will simplify the fix. Sorry for the step back, @davisaints. :grin: 

Thanks